### DISCUSSION TITLE
Change / remove "select an item" placeholder from select

### DISCUSSION THREAD

**Discussion by andreasCampeon** [adobe/react-spectrum#5206] [Q&A] [answered]
Hello all, I am using the `select` component and I am trying to render a select with no pre-selected item. However even on the documentation a 'default / placeholder' item is added ("Select an item").

Is there a way to remove it?

**Comment by reidbarber** [comment, marked as answer]
You should be able to use the `placeholder` prop to override this default. Example: https://codesandbox.io/s/friendly-grothendieck-5kpy6t

**Reply by brumm** [reply]
I ran into this recently and was surprised at the behavior. I did not expect react-aria to default to a hardcoded placeholder text. I was really confused because I had copied the code from an example and could not figure out where the text came from.

I went to the docs which don't call this out on the Select page. The placeholder prop default value is documented as empty

![CleanShot 2025-03-15 at 14 10 08](https://github.com/user-attachments/assets/478660da-2b40-4fce-b9f6-3355619007fb)

**Reply by snowystinger** [reply]
That looks like a mistake in the documentation, thanks for pointing it out.
I've addressed it here [fix: documentation select placeholder default](https://github.com/adobe/react-spectrum/pull/7941#top)
